packages/gitbook-docs/src/__tests__/parseContent.test.ts

```diff
@@ -255,4 +255,20 @@ describe('parseContent', () => {
       ]),
     ]);
   });
+
+  it('should parse embed', () => {
+    const embed =
+      '{% embed url="https://www.pagopa.it/" %}\n' +
+      'This is a caption\n' +
+      '{% endembed %}';
+    expect(parseContent(embed, config)).toStrictEqual([
+      new Markdoc.Tag(
+        'Embed',
+        {
+          url: 'https://www.pagopa.it/',
+        },
+        [new Markdoc.Tag('Paragraph', {}, ['This is a caption'])]
+      ),
+    ]);
```

Comment:
Can you add a test that covers the [self-closed tag scenario](https://github.com/pagopa/devportal-docs/blob/docs/from-gitbook/docs/v7eFccIIRcGfOIG6IHkT/esperienza-per-il-cittadino/app-io/README.md?plain=1#L22)?